### ISSUE TITLE
sequence padding quantizes embedding

### ISSUE THREAD

**Issue by compartia** [nemoware/analyser#170]
keras.preprocessing.sequence pad_sequences makes embedding int-descrete
see picture
<img width="1020" alt="Capture d’écran 2020-05-26 à 10 36 56" src="https://user-images.githubusercontent.com/7677437/82855173-2c1ad980-9f3d-11ea-82f3-b6657a96fb70.png">
